app/src/main/java/com/jnj/vaccinetracker/participantflow/screens/ParticipantFlowMatchingViewModel.kt

```diff
@@ -212,6 +212,7 @@ class ParticipantFlowMatchingViewModel @Inject constructor(
                         participantId = participant.participantId,
                         irisMatchingScore = participant.matchingScore,
                         birthDateText = participant.birthDate.toDateTime().format(DateFormat.FORMAT_DATE),
+                        isBirthDateEstimated=participant.isBirthDateEstimated,
```

Comment:
Are all these changes part of this commit.

Reply:
yes